Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.py

```diff
@@ -1272,6 +1574,8 @@ def update_remote_system_command(client: Client, args: Dict[str, Any]) -> str:
                         args['entries']: the entries to send to the remote system
                         args['incident_changed']: boolean telling us if the local incident indeed changed or not
                         args['remote_incident_id']: the remote incident id
+        repository: Azure DevOps repository.
+        project: Azure DevOps project.
```

Comment:
is this the name of the repo? the id of the project? this should be clear to future maintainers.

Consider `The Azure DevOps repository name`/`id`/ ...